samples/bluetooth/central_path_loss_monitoring/src/main.c

```diff
@@ -0,0 +1,235 @@
+#include <bluetooth/scan.h>
+#include <dk_buttons_and_leds.h>
+#include <sdc_hci_vs.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/hci_vs.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/printk.h>
+
+static struct bt_conn *default_conn;
+
+/* Creating a delayed work item */
+static struct k_work_delayable tx_rssi_work;
+
+#define POLLING_INTERVAL_MS 500
+#define BAUDRATE_2M 2
```

Comment:
Usually we use "phy" when referring to baudrate in BLE. Could we update it to "2M_PHY"